src/Engine/GameInstance.java

```diff
@@ -198,7 +201,11 @@ public GameEventQueue turn()
 
     // Set the cursor to the new CO's last known cursor position.
     setCursorLocation(playerCursors.get(activeCoNum).xCoord, playerCursors.get(activeCoNum).yCoord);
-    
+
+    // Handle income and any other scenario-specific events.
+    events.addAll(gameScenario.initTurn(gameMap));
```

Comment:
Does `initTurn()` need the current CO or something, so it can calculate and dispense income?
Then again, maybe it's the better part of valour to not move stuff around too much yet?

Reply:
Might put that there eventually. Not yet though.